content/articles/how-to-use-lambdas-to-manipulate-through-a-collection-of-objects/index.md

```diff
@@ -0,0 +1,372 @@
+
+### INTRODUCTION
+Java is majorly an object-oriented programming language. The importance of lambdas as a programmer is to
+make your code appear shorter, efficient, and above all functional. Everything in java is an object. 
+
+An object could be physical or imaginary, it just has to do with the way we see the thing about things in general. For example, a wallet is an object that has a method to store money, make money, add cards, get cards, etc.
+<!--more-->
+Example 2: a bucket is an object that has a method to store fluids either solid, liquid, or gas depending on what you want to store it with and get fluid.
+
+Now it gets interesting when you have a collection of wallets with different items in it and you want to access the one with the red card that has money more than #30,000 ('#' is naira in Nigeria currency), this is when lambdas come in because it hard you iterate through a collection without making mistakes and even if it works it's definitely not efficient.
+
+
+### Prerequisite
+- A basic understanding of the Java OOP(Object Oriented Programming) concept
+- Basic understanding of lambdas
+
+### Objectives
+
+1. To enable individuals to understand how lambdas work
+1. To enable individuals to understand how to sort
+1. To enable individuals to understand how to calculate the value of the different object
+1. To enable individuals to understand how to Group a collection of object 
```

Comment:
Group should have a capitial G. Also, fix your Objectives numbering. Many of your numbered lists have this error.

Reply:
I think that is the way to make a list in an`.md`  file. I used virtual studio code and then use this shortcut (ctrl +k+v) to view it, should I list use normal numeric values.
please get back to me.

Reply:
Sorry, should I use the normal numeric value to list my objectives?

Reply:
1, 2, 3, 4 etc. Not all 1.